dist/css/bootstrap.css

```diff
@@ -6507,6 +6507,19 @@ tbody.collapse.in {
   right: auto;
   left: 0;
 }
+@media (max-width: 576px) {
+  .dropdown-menu-bottom {
+    position: fixed;
+    top: auto;
+    bottom: 0;
+    width: 100%;
+    max-width: 100% !important;
```

Comment:
Do we need both `width` and `max-width`?

Reply:
Can do without the width but inherits a calculated value for max-width that needs to be overwritten